spring-boot/admin/src/main/resources/application.yml

```diff
@@ -25,7 +26,14 @@ ygg:
     verification.server.url: http://localhost:8080
 
   admin:
-    database.insertSampleDataOnStartup: true
+    database:
+      # Define if you want to use development test data to be generated by CoatRack on startup
+      insertSampleDataOnStartup: true
```

Comment:
To be discussed: we should probably set this to "false" for the default config. For someone new to CoatRack that data will probably be confusing.